### DISCUSSION TITLE
How to find CIDR / Network range of target

### DISCUSSION THREAD

**Discussion by c4ir0** [projectdiscovery/tlsx#88] [Q&A] [answered]
excsuse me 

in this example [https://github.com/projectdiscovery/tlsx#sancn-probe](url)

how did U know that `173.0.84.0/24` is related to paypal I mean ... to get the IP range of some host or company

**Comment by ehsandeep** [comment, marked as answer]
https://github.com/projectdiscovery/asnmap

**Reply by c4ir0** [reply]
> https://github.com/projectdiscovery/asnmap

**I really started to ❤ u**
